### PR TITLE
KarinaKlevzhits/hometask-5

### PR DESCRIPTION
**Pull request by KarinaKlevzhits**

### CODE REVIEW

**Comment by Mat1s** [comment]
Can you insert empty line in the end of each files and insert ';' after queries?

**Comment by Mat1s** [comment]
Also please rewrite queries with statements on new line. It helps to become queries more readable.

File: KarinaKlevzhits/hometask_5/#4.11.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+  ALTER TABLE film DROP column income
```
Comment (Mat1s) [inline KarinaKlevzhits/hometask_5/#4.11.sql]: You should drop the same column that you added in previous request

File: KarinaKlevzhits/hometask_5/#4.8.sql [hunk 2]
```diff
@@ -0,0 +1,6 @@
+DELETE FROM film_category WHERE film_id in (SELECT film_id FROM film WHERE title = 'Aladdin Calendar');
+DELETE FROM film_actor WHERE film_id in (SELECT film_id FROM film WHERE title = 'Aladdin Calendar');
+DELETE FROM payment WHERE rental_id in (SELECT rental_id FROM rental WHERE inventory_id in (SELECT inventory_id FROM inventory WHERE film_id in (SELECT film_id FROM film WHERE title = 'Aladdin Calendar')));
+DELETE FROM rental WHERE inventory_id in (SELECT inventory_id FROM inventory WHERE film_id in (SELECT film_id FROM film WHERE title = 'Aladdin Calendar'));
+DELETE FROM inventory WHERE film_id in (SELECT film_id FROM film  WHERE title = 'Aladdin Calendar');
+DELETE FROM film WHERE title='Aladdin Calendar';
```
Comment (Mat1s) [inline KarinaKlevzhits/hometask_5/#4.8.sql]: Can you try to change tables with foreign key to delete all related record using only one request
alter table film_actor
  drop constraint film_actor_foreign_for_film_id_example,
  add constraint fim_id_fkey
    foreign key (film_id)
    references film (film_id)
    on delete cascade
    not valid;

File: KarinaKlevzhits/hometask_5/#4.3.sql [hunk 3]
```diff
@@ -0,0 +1,2 @@
+SELECT CONCAT(last_name,'-',first_name) FROM actor 
+WHERE first_name LIKE 'W%' OR last_name like '%H'
```
Comment (Mat1s) [inline KarinaKlevzhits/hometask_5/#4.3.sql]: Please make uppercase second 'like' to be consistent with all statements

**Comment by Mat1s** [comment]
It seem that you forgot to commit task 4.4

**Review by Mat1s — APPROVED** [review]
It's so cute, thanks!

**Comment by KarinaKlevzhits** [comment]
Great! Thanks for review))

**Pull request merged**